frontend/src/app/services/event-list.ts

```diff
@@ -31,6 +30,30 @@ function probability(probabilityRate: number){
   return probabilityRate > Math.random();
 }
 
+/**
+ * Returns true if EventMitigation is active
+ * @param eventInput - EventInput
+ * @param id - Id of the EventMitigation
+ */
+function isEventMitigationActive(eventInput: EventInput, id: string) {
+  const mitigation = eventInput.eventMitigations.find(em => em.id === id);
+  return !isNil(mitigation);
+}
+
+/**
+ * Callback that is called every day before trigger conditions are evaluated
+ * @param eventInput - EventInput
+ * @param id - Id of the EventMitigation
```

Comment:
Does not seem so.

Reply:
It is called from EventHandler.evaluateDay before the event triggers are evaluated

Reply:
I just meant the param documented is not in the function declaration below.